Mathlib/Probability/Moments.lean

```diff
@@ -336,4 +344,27 @@ theorem measure_le_le_exp_cgf [IsFiniteMeasure μ] (ε : ℝ) (ht : t ≤ 0)
 
 end MomentGeneratingFunction
 
+theorem integrable_bounded [IsFiniteMeasure μ] (a b : ℝ) {X : Ω → ℝ} (hX : AEMeasurable X μ)
```

Comment:
A better name would be `integrable_of_mem_Icc`.
This lemma should be moved to a more appropriate place: it is about integrability in general, and not about moments. You can use `#find_home! integrable_of_mem_Icc` to suggest a more suitable file. Probably Mathlib.MeasureTheory.Function.L1Space